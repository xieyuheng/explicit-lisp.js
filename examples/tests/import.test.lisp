(import "compose.lisp" id compose (rename compose c))

;; (assert-equal
;;   (compose
;;    (compose id id)
;;    (compose id id))
;;   (c (c id id) (c id id))
;;   id)

(compose
 (compose id id)
 (compose id id))

(c (c id id) (c id id))

id
