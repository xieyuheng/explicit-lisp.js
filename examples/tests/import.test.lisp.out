(lambda (x₄) x₄)
(lambda (x₁₀) x₁₀)
(lambda (x) x)
